Scripts/Miscellaneous/Arrow_Angle_detector/Arrow.py

```diff
@@ -0,0 +1,107 @@
+import cv2
+import numpy as np
+
+
+def nothing(x):
+	pass
+
+font = cv2.FONT_HERSHEY_COMPLEX
+cap = cv2.VideoCapture(0)
+cv2.namedWindow("Adjust")
+cv2.createTrackbar("min","Adjust",110,255,nothing)
+#resized = cv2.resize(image, (width, height), interpolation=cv2.INTER_AREA)
+text = 'Left'
+while True:
+	ret ,frame = cap.read()
+	frame = cv2.resize(frame, (640,480), interpolation=cv2.INTER_AREA)
+	cropped = frame[100:400,100:500]
+	cropped = cv2.flip(cropped,1)
+	temp = cropped
+	cropped = cv2.cvtColor(cropped, cv2.COLOR_BGR2GRAY)
+	mi = cv2.getTrackbarPos("min","Adjust")
+
+	_,threshold = cv2.threshold(cropped,mi,255,cv2.THRESH_BINARY)
+	kernal = np.zeros([10,10],np.uint8)
+	threshold = cv2.erode(threshold,kernal)
+	contours,_=cv2.findContours(threshold,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)
+
+	for cnt in contours :
+		area = cv2.contourArea(cnt)
+		if area > 400 :
+			approx = cv2.approxPolyDP(cnt,0.01*cv2.arcLength(cnt,True),True)
+			approx_area = cv2.contourArea(approx)
+			if(len(approx)==7):
+					n = approx.ravel()
+					x1 = n[0]
+					y1 = n[1]
+					x2 = n[2]
+					y2 = n[3]
+					x3 = n[6]
+					y3 = n[7]
+					distance1 = (x1 - x2)*(x1 - x2) + (y1 - y2)*(y1 - y2)
+					distance2 = (x1 - x3)*(x1 - x3) + (y1 - y3)*(y1 - y3)
+					ratio = distance1/distance2
+					if(2500<approx_area<25000 and (0.2<ratio<0.3 or ratio <0.1)):
+						cv2.drawContours(temp,[approx],0,(0,0,255),5)
+
+						x = approx.ravel()[0]
+						y = approx.ravel()[1]
+						
+						#x1 = n[0]
+						#y1 = n[1]
+						#x2 = n[2]
+						#y2 = n[3]
+						#x3 = n[6]
+						#y3 = n[7]
+						#distance1 = (x1 - x2)*(x1 - x2) + (y1 - y2)*(y1 - y2)
+						#distance2 = (x1 - x3)*(x1 - x3) + (y1 - y3)*(y1 - y3)
```

Comment:
Remove unnecessary commented out code lines, add comments instead for better understanding,